manuscript/figure_captions/figure_1_caption.md

```diff
@@ -0,0 +1 @@
+![** tHMM influence diagram for $𝑵$ cells with $𝑲$ states. The latent states ($\bm{z}$) represent subpopulation classification of each cell. The observed states ($\bm{x}$) in blue/orange denote measured phenotypes, particularly fitness (i.e. end-of-life fate by binary outcome variable $\phi$, and lifetime by variable $\tau$ in hours). The tHMM is robust to cells that die, such as cell $6$.. ...(./Figures/figure2.svg){#fig:tfac}
```

Comment:
Figures should be included within the text approximately where they should appear.

Reply:
Also, I don't think you ever close your `[` bracket?

Reply:
![**Overall title. ** A) blah.  B) blah blah ...(./Figures/figure3.svg){#fig:tfac}

This is the example format i was told to use ....

Can you give me the way to input figure captions? Where do I close the bracket? After title or whole text

Reply:
i put it after the title. still failes